docs/assets/schemas/api-latest.yaml

```diff
@@ -284,6 +284,30 @@ paths:
                     type: string
                     description: Reason for the failure
           description: ''
+  /api/notifications/:
```

Comment:
By running the tests the API schema and redoc docs get updated with thr new endpoint 👌